tapdb/universe_perf_test.go

```diff
@@ -191,10 +176,10 @@ func TestUniverseIndexPerformance(t *testing.T) {
 	t.Parallel()
 
 	const (
-		numAssets         = 25
+		numAssets         = 50
```

Comment:
We might want to have a testing flag in which we can manually significantly bump these numbers to see how the tradeoffs work out at scales more close to what we see in our main universe.

Reply:
I went ahead and created a `universe-scale` testing flag. I'm not sure if this is the correct approach here